src/containers/pages/FocusInvestigation/single/index.tsx

```diff
@@ -232,10 +222,7 @@ class SingleFI extends React.Component<RouteComponentProps<RouteParams> & Single
       [currentReactivePlansArray, currentRoutinePlansArray].forEach((plansArray: Plan[]) => {
         if (plansArray && plansArray.length) {
           const thePlans = plansArray.map((item: Plan) => {
-            let thisItem = extractPlan(item);
-            // transform values of this properties if they are null
-            const columnsToTransform = ['village', 'canton', 'district', 'province'];
-            thisItem = transformValues(thisItem, columnsToTransform);
+            const thisItem = extractPlan(item);
             return thisItem;
```

Comment:
Same here, since the extra logic was moved out of this block, it seems like this could be a one-liner.